BeeJet.Bot/Commands/Handlers/GameManagement/GameButtonsPressedHandler.cs

```diff
@@ -51,26 +60,19 @@ public async Task LeaveGamePressed()
             await Context.ComponentInteraction.DeferAsync();
         }
 
-        private static bool TryGetGameName(IUserMessage message, out string gameName)
+        private bool TryGetChannelId(IUserMessage message, string customButtonId, out ulong channelId)
```

Comment:
Zou deze niet op een centralere plek moeten ? Zal vast niet de enige plek zijn waar je op basis van iets een channelId wilt bepalen

Reply:
Maar hier bepaal je heel specifiek hoe je het moet ophalen. Dus wat voor context heb je opgeslagen en hoe kom ik aan het ID van die context. De gedeelde code zit in GetButtonContextForMessageIdAndCustomId. Enige code die dan gelijk zou zijn is de if null, maar dat lijkt me weer niet zo belangrijk om naar 1 plek te trekken?

Vermoed dat er ook niet nog een extra plek gaat komen die hetzelfde doet, denk dat dan snel de HandlerContext heel specifiek is voor een bepaalde situatie en van een bepaalde klasse is ipv een enkele string.